README.md

```diff
@@ -61,26 +75,27 @@ clickDialogNeutralButton();
 clickDialogNegativeButton();
 
 // Scroll on scrolls and pagers
-scrollTo(R.id.button);
+scrollTo(R.id.far_away_widget);
 scrollTo("A widget with this text");
 swipeViewPagerForward();
-swipeViewPagerForward(R.id.pager);
 swipeViewPagerBack();
-swipeViewPagerBack(R.id.pager);
 
 // Interact with the navigation drawer
-openDrawer(R.id.drawer);
-closeDrawer(R.id.drawer);
+openDrawer();
+closeDrawer();
+openDrawerWithGravity(Gravity.RIGHT);
+closeDrawerWithGravity(Gravity.RIGHT);
 
```

Comment:
Another MIMIMI, but what about putting together the two `openDrawer` and `closeDrawer`?